outsystems/devops.md

```diff
@@ -1,12 +1,22 @@
 # Outsystems DevOps Standards
 
-##
+## Deployment
 
+### Cyclic Dependencies
+A cyclic dependency is formed when two or more modules have direct or indirect dependencies on each other.
 
-## Deployment
+A feature of the [discovery](https://lgss-dev.outsystemsenterprise.com/Discovery/) web application is to identify where cyclical dependencies have been created so that the abstraction can be rectified.
 
-### Deployment Messages
+**Further reading:** [Validating your application architecture](https://success.outsystems.com/Support/Enterprise_Customers/Maintenance_and_Operations/Designing_the_architecture_of_your_OutSystems_applications/03_Validating_your_application_architecture)
+
+### Solutions
+Before deploying it's important to make sure that all references are up to date. A simple way to achieve this is by creating and maintaining a solution. A solution enables outsystems to handle updating modules in the correct order.
 
+The naming convention currently us for solutions is `Application_{name of application}`
```

Comment:
Spelling error on row 15. 'The naming convention currently us'. Assumption is that it should be 'The naming convention currently used'

Reply:
@ScottyTooHotty Thanks, I believe if fixed this now 👍